protocol/0069-VCBS-validators_chosen_by_stake.md

```diff
@@ -158,6 +184,8 @@ See [limited network life spec](./0073-LIMN-limited_network_life.md).
 |`network.validators.multisig.numberOfSigners`              | String (integer) |       9       | Currently set to the number of validators on the network. In future will be used to scale multisig Validator participation.  |
 |`network.validators.ersatz.rewardFactor`                   | String (float)   |      0.2      | Scales down [the rewards](./0069-VCBS-validators_chosen_by_stake.md#ersatz-validators) of ersatz validators relative to actual validators  |
 |`network.validators.ersatz.multipleOfTendermintValidators` | String (integer) |       2       | Used to [calculate the number](./0069-VCBS-validators_chosen_by_stake.md#ersatz-validators) of ersatz Validators that will earn rewards |
+|`spam.protection.minMultisigUpdates`                       | String (integer) |       5       | Minimum number of staked tokens a party must have to be able to submit a transaction to issue signatures |
+|`spam.protection.max.MultisigUpdates`                      | String (integer) |       5       | Maximum number of times per epoch a party can submit a transaction to issue signatures |
```

Comment:
Note that the `min` one already exists but was missing.

Currently there is no network parameter to control the max-number of issue-signature transactions per epoch and so it defaults to one. This has already caused minor annoyance since when a "swap" happens in the validator set you need to send in two transactions, one to get the add signatures, and one to get the remove.

With a second bridge, this now becomes 4 transactions, 2 for each bridge. So we need this as network parameter really.

Reply:
I think that's fine and you've added the max - happy to call this resolved @wwestgarth ?